src/rgw/rgw_asio_frontend.cc

```diff
@@ -370,6 +374,24 @@ static int drop_privileges(CephContext *ctx)
   return 0;
 }
 
+template <typename Iter>
+int AsioFrontend::process_ports(const Iter& begin,const Iter& end,
+				ip_proto_t p, bool use_ssl)
+{
+  boost::system::error_code ec;
+  for (auto it=begin; it != end; it++) {
```

Comment:
missing spaces around =. use preincrement for iterators

Reply:
ack